server/src/main/java/com/sauron/service/UserServiceImpl.java

```diff
@@ -48,17 +49,19 @@ private UserView convertToUserView(User user) {
         userView.setId(user.getId());
         userView.setEmail(user.getEmail());
         userView.setUsername(user.getUsername());
-        userView.setBanks(user.getBankAccounts().stream()
+        userView.setBanks(user.getBankConnectionData().stream()
+                .map(BankConnectionData::getBank)
                 .map(this::convertToBankView)
                 .collect(Collectors.toList()));
```

Comment:
Now I see that if we allow multiple connections per bank, we probably should limit this stream to distinct elements to avoid duplication of banks on UI.